deploy/crds/operator.ibm.com_grafanas_crd.yaml

```diff
@@ -186,11 +169,6 @@ spec:
                 enabled:
                   type: boolean
               type: object
-            prometheusServiceName:
```

Comment:
instead of removing attributes, keep them optional and deprecated

Reply:
@pgodowski looks like all of the removed attributes are optional, reverted attribute removal changes in crd, csv and cr
Could you please check now